clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/ElasticSearchEntityTagger.java

```diff
@@ -231,23 +257,34 @@ private static UpsertEntityTagsDescription upsertEntityTagsDescription(String ty
                                                                          String key,
                                                                          String value,
                                                                          Long timestamp) {
+
+    String name = prefix + key;
+    Map<String, String> tagValue = new HashMap<>();
```

Comment:
This tag value structure is very specific to the notice/alerts ... consider renaming the method to something more specific than `upsertEntityTagsDescription`?